### ISSUE TITLE
MAX - Editing zones is a pain, esp. at higher altitudes.

### ISSUE THREAD

**Issue by harkinhails** [TheGameCreators/GameGuruRepo#5322]
Selecting vertices to edit zones at higher levels (non flat surfaces) results in the object disappearing inside the terrain, especially since vertices snap to the terrain.

![vertices_01](https://github.com/TheGameCreators/GameGuruRepo/assets/102403572/7991cb54-5e82-412b-9137-b500c0706c92)

Only way to edit it is to move the zone up in the air, edit it in top down view and the move it to the location. Repeat the process when you need to re-edit the zone later.

![vertices_02](https://github.com/TheGameCreators/GameGuruRepo/assets/102403572/e93ccfa4-e3e7-4176-8ca3-983495c64198)

While we're at it, this issue can be dealt with by just allowing the "positioning mode" to lock the vertices at specific axis, allowing easy edits in the place. Giant arrows like in Max/Maya to move the vertices would be helpful in positioning them around terrain.

Positioning mode only seems to affect the zone object all over, not the vertices (maybe a toggle?).

Actually, just adding giant arrows to move the vertices would be a great help (arrows that can be increased or decreased in size using +- keys, like in Max).

**Comment by MonkeyFrogStudio** [comment]
I agree. Resizing zones are a pain anywhere, let alone on terrain, etc. I had recommended at least a gizmo to move the points about here:

https://github.com/TheGameCreators/GameGuruRepo/issues/4362

**Comment by davetgc** [comment]
I'm putting this down as an enhancement as I agree the zones do need revisiting, but they do work (Mostly).
